helpers/useProcessSectionComplete.ts

```diff
@@ -16,7 +16,13 @@ const useProcessSectionComplete = <
   S extends StoreNames<ProcessDatabaseSchema["schema"]>
 >(
   processRef: ProcessRef,
-  storeNames: S[]
+  storeNames: S[],
+  validator?: (
+    values: StoreValue<
+      ProcessDatabaseSchema["schema"],
+      StoreNames<ProcessDatabaseSchema["schema"]>
```

Comment:
```suggestion
      S
```

We know the types because we use a generic to defined the store names we pass in.